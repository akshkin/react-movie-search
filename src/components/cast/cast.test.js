import { render, screen } from "@testing-library/react"
import Cast from "./cast.component"

test("should render Cast correctly", () => {
    const mockChar = {
        profile_path: "https://image.tmdb.org/t/p/w500/poster", 
        original_name: "Morgan Freeman", 
        character: "Ellis Boyd"  
    } 
    const {snapshot} = render(
            <Cast char={mockChar} baseImg="https://image.tmdb.org/t/p/w500"/>        
    )
    expect(snapshot).toMatchSnapshot()
})