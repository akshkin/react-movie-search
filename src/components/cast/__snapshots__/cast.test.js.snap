// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Cast correctly 1`] = `undefined`;
